### PR TITLE
Allow scsb full dumps to index

### PR DESCRIPTION
**Pull request by hackartisan**
refs #1382

### CODE REVIEW

File: spec/models/dump_file_spec.rb [hunk 1]
```diff
@@ -28,11 +28,16 @@
...
+    let(:recap_record_full_type) { DumpFileType.find_by(constant: 'RECAP_RECORDS_FULL') }
     let(:other_type) { DumpFileType.create }
     it 'returns true for RECAP_RECORDS type' do
       df = DumpFile.new(dump_file_type: recap_record_type)
       expect(df.recap_record_type?).to eq true
     end
+    it 'returns true for RECAP_RECORDS_FULL type' do
+      df = DumpFile.new(dump_file_type: recap_record_full_type)
+      expect(df.recap_record_type?).to eq true
+    end
```
Comment (christinach) [inline spec/models/dump_file_spec.rb:40]: should this be `expect(df.recap_record_full_type?).to eq true` ?

File: docs/indexing.md [hunk 2]
```diff
@@ -57,3 +57,18 @@ as deploy user, in `/opt/marc_liberaton/current`
...
 
 This step takes about 21 hours
+
+### Index Partner SCSB records
+
+If needed, pull the most recent SCSB full dump records into a dump file:
+
+- as deploy user, in `/opt/marc_liberaton/current`
+- `$ RAILS_ENV=production bundle exec rake scsb:import:full
+- It kicks off an import job
```
Comment (christinach) [inline docs/indexing.md:67]: can you add a ` at the end of line $ RAILS_ENV=production bundle exec rake scsb:import:full

**Review by christinach — CHANGES_REQUESTED** [review]

File: spec/models/dump_file_spec.rb (hunk 1 above)
Comment (hackartisan) [reply spec/models/dump_file_spec.rb:40]: That's the name of the method; I extended its use to cover the new recap dump type as well.

File: docs/indexing.md (hunk 2 above)
Comment (hackartisan) [reply docs/indexing.md]: yes thanks for catching this -- done.

**Review by christinach — APPROVED** [review]

**Pull request merged**